### PR TITLE
Change wording of "review" to "approval"

### PR DESCRIPTION
**Pull request by abastola0**
Fixes: #2554

### CODE REVIEW

**Review by MizukiTemma — CHANGES_REQUESTED** [review]
Thank you so much for your contribution 👍 I would like to suggest some further adjustment.

1. Add a release note
  like the other notes in integreat-cms/integreat_cms/release_notes/current/unreleased/
2. Replace "Pending Review" into "Pending approval" in 
  integreat-cms/integreat_cms/cms/constants/status.py
  integreat-cms/integreat_cms/cms/templates/translations/translations_management.html
  integreat-cms/integreat_cms/cms/views/pages/page_form_view.py
  Then your change in `django.po` comes into force :)
3. Use ./tools/translate.sh and ./tools/check_translations to update the translation file

Feel free to ask whatever if you have a question ;)

**Comment by MizukiTemma** [comment]
Thank you for speedy updates 😃 

Only migration is missing: you can use `./tools/migrate.sh` and after that `./tools/black.sh`

**Comment by abastola0** [comment]
> Thank you for speedy updates 😃
> 
> Only migration is missing: you can use `./tools/migrate.sh` and after that `./tools/black.sh`

Done. I'm pushing all my updates to single commit if that's not a problem.

File: integreat_cms/cms/migrations/0084_auto_20231129_1607.py
Comment (MizukiTemma) [inline integreat_cms/cms/migrations/0084_auto_20231129_1607.py:9]:
```suggestion
class Migration(migrations.Migration):
    """
    Change wording of "review" to "approval"
    """

    dependencies = [
        ("cms", "0083_add_page_based_offer_fields"),
    ]
```

Comment (MizukiTemma) [reply integreat_cms/cms/migrations/0084_auto_20231129_1607.py:9]: In addition, could you rename the file name to "0084_rename_review_to_approval.py" ?

**Review by MizukiTemma — CHANGES_REQUESTED** [review]

**Review by MizukiTemma — APPROVED** [review]
🎉

**Comment by abastola0** [comment]
@MizukiTemma Does this merge automatically? It says it requires 2 approving reviews.

**Comment by MizukiTemma** [comment]
@abastola0 
You need one more approval from one of us to mearge this PR since we do double-check, and I'd like to recommend not to activate auto-merge to avoid any potential problem.
I'll ping you when it becomes ready to merge so you can enjoy clicking "merge" button🙂

**Review by timobrembeck — CHANGES_REQUESTED** [review]
Thanks for your first PR with us :rocket: 

I found a few more occurrences of this wording and wondered whether we might also adapt it in the other places to stay consistent?

- [integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html#L20](https://github.com/abastola0/integreat-cms/blob/5ccd6e11327bdc8c34bc33c9ce382a6b3d743dd0/integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html#L20):
  "is waiting for a review" -> "is waiting for approval"

- [integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html#L24](https://github.com/abastola0/integreat-cms/blob/5ccd6e11327bdc8c34bc33c9ce382a6b3d743dd0/integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html#L24):
  "no page waiting for a review" -> "no page waiting for approval"

Also, there are a few occurrences where it might not be completely straight forward, e.g. [integreat_cms/cms/templates/pages/page_form.html#L67](https://github.com/abastola0/integreat-cms/blob/5ccd6e11327bdc8c34bc33c9ce382a6b3d743dd0/integreat_cms/cms/templates/pages/page_form.html#L67):
"Submit for review" -> "Submit for approval"? Or does this sound weird?

We can also ignore this for the moment and let our service team decide whether it's worth opening another issue.

And apart from that, could you change the German translation in [integreat_cms/locale/de/LC_MESSAGES/django.po#L4768](https://github.com/abastola0/integreat-cms/blob/5ccd6e11327bdc8c34bc33c9ce382a6b3d743dd0/integreat_cms/locale/de/LC_MESSAGES/django.po#L4768)
"Diese Revision wiederherstellen und zum Review vorlegen" -> "Diese Revision wiederherstellen und zur Überprüfung vorlegen"?
It's not completely what the issue description states, but I think it makes sense to at least remove the last occurrence of the word "Review" in the German translation, even if we don't have a consistent solution in the English text yet.

Thanks a lot in advance for taking another look! :pray:

File: integreat_cms/release_notes/current/unreleased/2576.yml
Comment (abastola0) [reply integreat_cms/release_notes/current/unreleased/2576.yml]: Hi @timobrembeck and @MizukiTemma. Any update on this PR?

File: integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html
Comment (timobrembeck) [inline integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html:20]:
```suggestion
            The page <b>{{ single_unreviewed_page }}</b> is waiting for approval. Please review the page to ensure the content is accurate.
```

File: integreat_cms/locale/de/LC_MESSAGES/django.po
Comment (timobrembeck) [inline integreat_cms/locale/de/LC_MESSAGES/django.po:4922]:
```suggestion
"The page <b>%(single_unreviewed_page)s</b> is waiting for approval. Please "
```

File: integreat_cms/release_notes/current/unreleased/2576.yml
Comment (timobrembeck) [reply integreat_cms/release_notes/current/unreleased/2576.yml]: Hi @abastola0 thanks for your changes! Sorry for the delay, we're a team of volunteers and part-time employees, so we can't always respond on the same day.

**Review by timobrembeck — CHANGES_REQUESTED** [review]
Awesome, thanks! Just a tiny detail missing, then I think this PR is good to go! :tada:

**Comment by abastola0** [comment]
> Awesome, thanks! Just a tiny detail missing, then I think this PR is good to go! 🎉

done!

File: integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html
Comment (PeterNerlich) [inline integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html:10]: What about including "with" when talking about pages? I'm not a native English speaker and I can't explain my gut feeling, but this sounds a bit more natural to me:

```suggestion
    {% translate "Pages with pending approval" %}
```

File: integreat_cms/locale/de/LC_MESSAGES/django.po
Comment (PeterNerlich) [inline integreat_cms/locale/de/LC_MESSAGES/django.po:4917]: The German translation could stay as it is
```suggestion
msgid "Pages with pending approval"
msgstr "Seiten, die zur Freigabe vorliegen"
```
…or could be adapted like the following. Nothing really changes meaningfully, it's merely more of a literal translation of the changed English phrase, and some might argue that it would be more elegant to not have a comma here:
```suggestion
msgid "Pages with pending approval"
msgstr "Seiten mit ausstehender Freigabe"
```

**Review by timobrembeck — APPROVED** [review]
@abastola0 I'm very sorry for taking so long to get back to this PR, but after resolving the git conflicts and fixing a few small other occurrences of the German wording, I think this is finally ready to be merged! :tada:

**Pull request merged**
